spec/venue_spec.rb

```diff
@@ -43,13 +40,67 @@
   end
 
   describe '#yell_at_patrons' do
+    # Test that the return patron string values are converted to upper case.
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
+
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capacity?' do
+    # Check if the patron list is under the capacity limit.
+    it 'checks at or below capacity limit' do
```

Comment:
Nice testing here. If you wanted to you could combine this test and the next test like such:
``` 
venue = Venue.new('Bluebird', 4)
venue.add_patron('Mike')
venue.add_patron('Megan')
venue.add_patron('Bob')

expect(venue.over_capacity?).to eq(false)

venue.add_patron('James')
venue.add_patron('Cat')

 expect(venue.over_capacity?).to eq(true)
```

Reply:
Yeah, I think originally I had it this way, but I broke it out for ease of understanding to someone later. I could have done it your way and just commented well though. After the discussion the other day, I started trying to make sure to limit testing scope for ease of understanding, but then there is the cost to length and time.